src/parachain/democracy.ts

```diff
@@ -0,0 +1,97 @@
+import { ApiPromise } from "@polkadot/api";
+import { DemocracyVote } from "@polkadot/types/lookup";
+
+import { addHexPrefix } from "../utils/encoding";
+import { TransactionAPI } from "..";
+
+/**
+ * @category InterBTC Bridge
+ */
+export interface DemocracyAPI {
+    /**
+     * @param proposalHash
```

Comment:
We should describe the parameters wherever possible, to do this I usually check the descriptions in `augment-api-tx.ts`